diff --git a/node_modules/eslint-plugin-require-js-extension/index.js b/node_modules/eslint-plugin-require-js-extension/index.js
index f0165e5..2b4465d 100644
--- a/node_modules/eslint-plugin-require-js-extension/index.js
+++ b/node_modules/eslint-plugin-require-js-extension/index.js
@@ -65,7 +65,8 @@ function rule(check) {
         const value = source.value.replace(/\?.*$/, '')
         if (!value || !value.startsWith('.') || value.endsWith('.js')) return
 
-        check(context, node, resolve(dirname(context.getFilename()), value))
+        const filename = 'getFilename' in context ? context.getFilename() : context.filename
+        check(context, node, resolve(dirname(filename), value))
       }
 
       return {
